apps/wpcom-block-editor/src/wpcom/features/add-tags-education-link.js

```diff
@@ -0,0 +1,33 @@
+import { localizeUrl } from '@automattic/i18n-utils';
+import { ExternalLink } from '@wordpress/components';
+import { createHigherOrderComponent } from '@wordpress/compose';
+import { addFilter } from '@wordpress/hooks';
+import { useTranslate } from 'i18n-calypso';
+import tracksRecordEvent from './tracking/track-record-event';
+
+const trackTagsEducationLinkClick = () =>
+	tracksRecordEvent( 'wpcom_block_editor_tags_education_link_click' );
+
+const addTagsEducationLink = createHigherOrderComponent( ( PostTaxonomyType ) => {
+	return ( props ) => {
+		const translate = useTranslate();
+
+		if ( props.slug !== 'post_tag' ) {
+			return <PostTaxonomyType { ...props } />;
+		}
+
+		return (
+			<>
+				<PostTaxonomyType { ...props } />
+				<ExternalLink
+					href={ localizeUrl( 'https://wordpress.com/support/posts/tags/' ) }
+					onClick={ trackTagsEducationLinkClick }
+				>
+					{ translate( 'Build your audience with tags' ) }
+				</ExternalLink>
+			</>
+		);
+	};
+}, 'addTagsEducationLink' );
+
+addFilter( 'editor.PostTaxonomyType', 'add-tags-education-link', addTagsEducationLink );
```

Comment:
I usually prefix filter ids with `wpcom` or `a8c` just to be extra sure. When we're writing code that runs in the editor we have to assume the user could have a number of 3rd party plugins installed so we have to be more defensive.

Docs for the `namespace` param talks about namespacing using slashes.
https://github.com/Aurorum/a-calypso-once-more/blob/f2307e1af07b94447e12d053ec6962f3f53efbc9/node_modules/@wordpress/hooks/src/createAddHook.js#L13

If we moved this code to the editing toolkit it's look something like:

```suggestion
addFilter( 'editor.PostTaxonomyType', 'a8c/full-site-editing/add-tags-education-link', addTagsEducationLink );
```

Reply:
I found that lots of namespaces are `full-site-editing/...` and the only one is `automattic/full-site-editing/...`. Maybe we can just use `full-site-editing/...` now?

Reply:
Yep that sounds good.